eldoc-box.el

```diff
@@ -578,6 +578,62 @@ display the docs in echo area depending on
     (when (eldoc-box--eldoc-message-function "%s" doc)
       (eldoc-display-in-echo-area docs interactive))))
 
+(defun eldoc-box--visible-p (&optional frame)
+  "Return non-nil if FRAME is visible."
```

Comment:
The docstring should probably mention that FRAME defaults to eldoc--frame.

Reply:
OK, I'll do it later.